js/demo-apps/packages/react-front-end/package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@lunasec/react-front-end",
   "private": true,
-  "version": "1.0.5",
+  "version": "1.0.6-alpha.1",
   "license": "Apache-2.0",
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "test:e2e": "cypress run --browser chrome",
-    "cypress:open": "cypress open --browser chrome",
+    "test:url": "if [ -z ${APPLICATION_FRONT_END+x} ]; then echo 'http://localhost:3000'; else echo $APPLICATION_FRONT_END; fi",
```

Comment:
in the future maybe we could run this kind of logic into our application code/test config.  I believe the cypress.js file can control this kind of thing programmatically..could be wrong though, not the end of the world either way